Sources/TuistGenerator/Linter/StaticProductsGraphLinter.swift

```diff
@@ -136,6 +148,14 @@ class StaticProductsGraphLinter: StaticProductsGraphLinting {
         ]
     }
 
+    private func shouldSkipDependency(_ dependency: GraphDependency, config: Config) -> Bool {
+        switch config.generationOptions.staticSideEffectsWarningTargets {
+        case .all: return true
+        case .none: return false
+        case let .only(names): return names.contains(dependency.name)
```

Comment:
Isn't this logic inverted? We should skip if `.none` or if not in `only`

Reply:
🤦🏼 Yes. I fixed it.